FinniversKit/Sources/Components/StatisticsView/StatisticsItemView.swift

```diff
@@ -4,37 +4,38 @@
 
 import UIKit
 
-public class StatisticsItemView: UIView {
+class StatisticsItemView: UIView {
```

Comment:
Is the class used in inheritance or can it be final?

Reply:
Good catch – adding